src/EnergyPlus/ChillerAbsorption.cc

```diff
@@ -951,7 +951,7 @@ void BLASTAbsorberSpecs::sizeChiller(EnergyPlusData &state)
     }
 
     if (PltSizNum > 0) {
-        if (state.dataSize->PlantSizData(PltSizNum).DesVolFlowRate >= state.dataHVACGlobal->TimeStepSys) {
+        if (state.dataSize->PlantSizData(PltSizNum).DesVolFlowRate >= HVAC::SmallWaterVolFlow) {
```

Comment:
Fix that @mjwitte had flagged here

Reply:
lololol vol flow > time step?  Maybe we should convert Real64 to a units-backed floating point type 

JUST KIDDING EVERYONE!